### DISCUSSION TITLE
Welcome to SSD365VSAddIn Discussions!

### DISCUSSION THREAD

**Discussion by shashisadasivan** [shashisadasivan/SSD365VSAddIn#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bolun-git** [comment]
Hello @shashisadasivan great product, I appreciate your effort on making this! I do have a question though we are recently start playing with the UDE environment and I notice I can install your solution for VS2022, but I can't installed it for UDE, just wondering if you have or planning to have your product updated with UDE environment? Thanks!

**Reply by shashisadasivan** [reply]
@Bolun-git Thankyou for your kind words.
At this stage i havent had the chance to work with a UDE environment yet. Once i get there i will definitely try to work on it !
If you want to DM bnme directly to collaborate on this, please feel free to do so at [LinkedIn](https://www.linkedin.com/in/shashisadasivan/)

**Reply by TrudAX** [reply]
Try to check this install logic -  https://github.com/TrudAX/TRUDUtilsD365/blob/master/InstallToVS/Program.cs . I don't have UDE either, but I remember someone told me that this install resolved after the latest changes

**Reply by Bolun-git** [reply]
@TrudAX I was able to create a C# console application by modifying the https://github.com/TrudAX/TRUDUtilsD365/blob/master/InstallToVS/Program.cs to @shashisadasivan DLLs and successfully install your Addins to UDE environment! 
![Screenshot 2025-03-06 133138](https://github.com/user-attachments/assets/8f228b38-9d14-459d-8fa1-a1f81a334bbb)

**Reply by shashisadasivan** [reply]
Amazing @Bolun-git 
I will try to put your suggestion into the front page.
Thanks a ton @TrudAX for that suggestion!
